schemas/ispyb/stored_programs/sp_retrieve_dc.sql

```diff
@@ -59,6 +62,9 @@ BEGIN
 					dc.beamSizeAtSampleY "beamSizeAtSampleY",
 					dc.focalSpotSizeAtSampleX "focalSpotSizeAtSampleX",
 					dc.focalSpotSizeAtSampleY "focalSpotSizeAtSampleY",
+					dc.POLARISATION "polarisation",
+					dc.actualCenteringPosition "actualCentringPosition",
```

Comment:
You've spelled it "actualCenteringPosition" in the gda-ispyb-api!

Reply:
I'll change it on the api side. "Centring" is apparently the preferred English spelling but I only found this out the other day. I've seen it spelled both ways in various places.

Reply:
OK, sounds good.